project/Game.py

```diff
@@ -122,10 +122,26 @@ def modify_landmark(self, oldlandmark, newlandmark):
         self.landmarks = [x.replace(oldlandmark, newlandmark) for x in self.landmarks]
 
     def set_point_penalty(self, points):
-        self.points = points
+        if not self.started:
+            try:
+                points = int(points)
```

Comment:
it's generally bad form to redefine an argument variable. You could skip the whole extra variable (and the else statement) by just directly setting self.penaltyValue.

Reply:
I guess the big thing I was trying to prevent was any input being a non-integer value and if it was then an exception would be thrown.  Is that considered a poor way of handling things in Python?

Reply:
no, that's fine, but the exception will be thrown before anything gets changed, so this would be equivalent (and simpler)
`        if not self.started:
            try:
                self.penaltyValue = int(points)
                return True
            except ValueError:
                return False`

Reply:
Ahh okay, gotcha.